### DISCUSSION TITLE
[조영재] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by youngjae-Jo** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#90] [2024년도 과제 제출]
![스크린샷 2024-07-17 17-02-38](https://github.com/user-attachments/assets/080361fb-ec96-4b17-9d60-ada950eab42f)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다.

설문조사는 과제에 포함된 필수 항목입니다. 아래 링크에서 설문을 완료하시기 바랍니다.
https://forms.gle/DDwJCEf9Zbs3fxFi8

감사합니다.
